README.md

```diff
@@ -1,96 +1,183 @@
+
+```javascript
+// index.js
+const { isValid, openAttestationVerifiers, verificationBuilder } = require("@govtechsg/oa-verify");
+const document = require("./document.json");
+
+const verify = verificationBuilder(openAttestationVerifiers, {
+  network: "ropsten",
+});
+
+verify(document).then((fragments) => {
+  console.log(isValid(fragments)); // output true
+});
+```
+
+### Custom verification
+
+By default the provided `verify` method performs multiple checks on a document
+
+- for the type `DOCUMENT_STATUS`: it runs `OpenAttestationEthereumDocumentStoreStatus` and `OpenAttestationEthereumTokenRegistryStatus` verifiers
+- for the type `DOCUMENT_INTEGRITY`: it runs `OpenAttestationHash` verifier
+- for the type `ISSUER_IDENTITY`: it runs `OpenAttestationDnsTxt` verifier
+
+All those verifiers are exported as `openAttestationVerifiers`
+
+You can build your own verify method or you own verifiers:
+
+```typescript
+import { verificationBuilder, openAttestationVerifiers } from "@govtechsg/oa-verify";
+
+// creating your own verify using default exported verifiers
+const verify = verificationBuilder(openAttestationVerifiers); // this verify is equivalent to the one exported by the library
+const verify = verificationBuilder([openAttestationVerifiers[0], openAttestationVerifiers[1]]); // this verify only run 2 verifiers
+
+// creating your own verify using custom verifier
+import { verificationBuilder, openAttestationVerifiers, Verifier } from "@govtechsg/oa-verify";
+const customVerifier: Verifier = {
+  skip: () => {
+    // return a SkippedVerificationFragment if the verifier should be skipped or throw an error if it should always run
   },
-  {
-    "name": "OpenAttestationDidSignedDocumentStatus",
-    "reason": {
-      "code": 0,
-      "codeString": "SKIPPED",
-      "message": "Document was not signed by DID directly"
-    },
-    "status": "SKIPPED",
-    "type": "DOCUMENT_STATUS"
+  test: () => {
+    // return true or false
   },
-  {
-    "data": [
-      {
-        "location": "example.tradetrust.io",
-        "status": "VALID",
-        "value": "0xe59877ac86c0310e9ddaeb627f42fdee5f793fbe"
-      }
-    ],
-    "name": "OpenAttestationDnsTxtIdentityProof",
-    "status": "VALID",
-    "type": "ISSUER_IDENTITY"
+  verify: async (document) => {
+    // perform checks and returns a fragment
   },
-  {
-    "name": "OpenAttestationDnsDidIdentityProof",
-    "reason": {
-      "code": 0,
-      "codeString": "SKIPPED",
-      "message": "Document was not issued using DNS-DID"
-    },
-    "status": "SKIPPED",
-    "type": "ISSUER_IDENTITY"
+};
+
+// create your own verify function with all verifiers and your custom one
+const verify = verificationBuilder([...openAttestationVerifiers, customVerifier]);
+```
+
+Refer to [verification methods](https://www.openattestation.com/docs/developer-section/libraries/open-attestation-verify#custom-verification) to find out more on how to create your own custom verifier.
```

Comment:
page not found at the moment

Reply:
ok i fixed this. some of the links would still be broken as they are referencing the new route from the restructured oa documentation-website. the links here will be fixed as soon the oa document-website is updated to the new version